test_rft_csv_export_plugin is flaky

```
      assert output_file.read_text(encoding="utf-8") == dedent(

            """\
            Realization,Well,Case,Iteration,Pressure
            0,OBS,default,0,0.0
            1,OBS,default,0,1.0
            2,OBS,default,0,2.0
            """
        )

E           AssertionError: assert '' == 'Realization,...fault,0,2.0\n'
E             - Realization,Well,Case,Iteration,Pressure
E             - 0,OBS,default,0,0.0
E             - 1,OBS,default,0,1.0
E             - 2,OBS,default,0,2.0


/home/runner/work/ert/ert/tests/unit_tests/gui/test_rft_export_plugin.py:115: AssertionError
```

Failure seems to be due to a race condition where the file exists but does not contain the data yet.

--- a/tests/unit_tests/gui/test_rft_export_plugin.py
+++ b/tests/unit_tests/gui/test_rft_export_plugin.py
@@ -112,11 +112,14 @@ def test_rft_csv_export_plugin_exports_rft_data(
         ][0]
         rft_plugin.trigger()
         qtbot.waitUntil(output_file.exists, timeout=20000)
-        assert output_file.read_text(encoding="utf-8") == dedent(
-            """\
+        qtbot.waitUntil(
+            lambda: output_file.read_text(encoding="utf-8")
+            == dedent(
+                """\
             Realization,Well,Case,Iteration,Pressure
             0,OBS,default,0,0.0
             1,OBS,default,0,1.0
             2,OBS,default,0,2.0
             """
+            )
         )

